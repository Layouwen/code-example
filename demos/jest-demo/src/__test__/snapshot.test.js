import { getConfig } from "../utils";
describe("snapshot", () => {
  test("config", () => {
    expect(getConfig()).toMatchSnapshot();
  });

  test("inline snapshot", () => {
    expect({
      host: "localhost",
      port: 3001,
    }).toMatchInlineSnapshot(`
      {
        "host": "localhost",
        "port": 3001,
      }
    `);
  });
});
