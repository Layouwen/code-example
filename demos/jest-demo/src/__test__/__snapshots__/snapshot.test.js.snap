// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot config 1`] = `
{
  "host": "127.0.0.1",
  "port": 3000,
}
`;
